library/Icinga/Web/Navigation/Renderer/RecursiveMenuNavigationRenderer.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/* Icinga Web 2 | (c) 2025 Icinga GmbH | AGPLv3 */
+
+namespace Icinga\Web\Navigation\Renderer;
+
+/**
+ * Renderer for the multi level navigation in the sidebar menu
+ *
+ * @method NavigationRenderer getInnerIterator() {
+ *     {@inheritdoc}
+ * }
+ */
+class RecursiveMenuNavigationRenderer extends RecursiveNavigationRenderer implements NavigationRendererInterface
+{
+    public function beginChildren(): void
+    {
+        parent::beginChildren();
+
+        $currentItemName = $this->getInnerIterator()->current()->getParent()->getName();
+
+        if ($this->getDepth() + 1 > 1) {
+            $this->content[] = '<li class="nav-item nav-item-header">'
+                . '<p><i aria-hidden="true" class="icon-letter"></i> ' . $currentItemName . '</p>'
+                . '</li>';
```

Comment:
Please render this the usual way. (Like it's done in method render already.)
i.e. not *by hand*